api/api-common/src/main/kotlin/io/qalipsis/api/steps/TimerMeterStepSpecification.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * QALIPSIS
+ * Copyright (C) 2025 AERIS IT Solutions GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package io.qalipsis.api.steps
+
+import io.micronaut.core.annotation.Introspected
+import io.qalipsis.api.context.StepContext
+import io.qalipsis.api.meters.Timer
+import io.qalipsis.api.meters.specification.ComparableValueFailureSpecification
+import io.qalipsis.api.meters.TimerFailureConditionSpec
+import io.qalipsis.api.meters.TimerFailureConditionSpecImpl
+import java.time.Duration
+
+/**
+ * Specification for a [io.qalipsis.core.factory.steps.meter.TimerStep].
+ *
+ * @author Francisca Eze
+ */
+@Introspected
+data class TimerMeterStepSpecification<INPUT>(
+    val meterName: String,
+    val percentiles: Map<Double, Duration>,
```

Comment:
Why do you need the threshold here? Shouldn't it be specified by the failure spec?